Entities/User.cs

```diff
@@ -1,9 +1,21 @@
-
+using AspNetCore.Identity.MongoDbCore.Models;
 
 namespace FRITeam.Swapify.Entities
 {
-    public class User : BaseEntity
+    public class User : MongoIdentityUser
     {
-        
+        public string Name { get; set; }
```

Comment:
UserName is already in IdentityUser class parent

Reply:
this is not UserName but First name